taf/tests/test_api/test_roles.py

```diff
@@ -264,6 +272,40 @@ def test_remove_role_when_keep_targets(auth_repo_path, roles_keystore):
     assert file_path.is_file()
 
 
+def test_lis_keys(auth_repo_path):
```

Comment:
```suggestion
def test_list_keys(auth_repo_path):
```

Reply:
This was fixed in one of the subsequent commits I think